crates/examples/infra/mod.rs

```diff
@@ -830,6 +970,7 @@ pub async fn main_entry_point<
         hotshot,
         &mut transactions,
         transactions_to_send_per_round as u64,
+        (transaction_size + 8) as u64,
```

Comment:
In the changeset above I see you added a comment to explain the `+8`. I presume that its use here is for the same purpose, however, since it's >300 lines below the initial comment, do you think it'd be worth adding a comment here? Or perhaps making a named constant like (`TIMESTAMP_BYTES_SIZE`)?

Reply:
Welcome to the team! I should add a comment here, the reason is that `transaction_size` in the config means appending this size of bytes to a default transaction with 8 bytes (see `create_random_transaction`).